presto-orc/src/main/java/com/facebook/presto/orc/metadata/ColumnStatistics.java

```diff
@@ -22,7 +26,10 @@
     private final StringStatistics stringStatistics;
     private final DateStatistics dateStatistics;
     private final DecimalStatistics decimalStatistics;
+    private final boolean hasNull;
```

Comment:
Let's not add this.  Nullability can be inferred from the difference between the number of rows in rowgroup/stripe/value and numberOfValues.  If they are different then columns has nulls.

Specifically, I don't want to use this because there were bugs in ORC/DWRF that set this incorrectly, and the inference trick always works.